test/run.test.ts

```diff
@@ -33,27 +40,59 @@ const travelAudience: AudienceDefinition = {
   id: 'travel_id',
   name: 'Travel Audience',
   ttl: TTL,
-  lookback: 10,
+  lookBack: 10,
   occurrences: 2,
   keywords: ['travel'],
 };
 
+const lookBackInfinityAudience: AudienceDefinition = {
+  id: 'look_back_infinity_id',
+  name: 'Look Back Audience',
+  ttl: TTL,
+  lookBack: 0,
+  occurrences: 2,
+  keywords: ['lb'],
+};
+
+const lookBackAudience: AudienceDefinition = {
+  id: 'look_back_id',
+  name: 'Look Back Audience',
+  ttl: TTL,
+  lookBack: 2,
+  occurrences: 2,
+  keywords: ['lb'],
+};
+
 const ONE_SPORTS_PAGE_VIEW: Array<PageView> = pageViewCreator(
   timeStampInSecs(),
   ['sport'],
   1
 );
+
 const TWO_SPORTS_PAGE_VIEW: Array<PageView> = pageViewCreator(
   timeStampInSecs(),
   ['sport'],
   2
 );
+
 const TWO_SPORTS_PAGE_VIEW_AFTER_TTL: Array<PageView> = pageViewCreator(
   timeStampInSecs() - TTL,
   ['sport'],
   2
 );
 
+const LOOK_BACK_INFINITY_PAGE_VIEW: Array<PageView> = pageViewCreator(
+  0,
```

Comment:
Are we setting this on purpose to show that anything will match?

Reply:
yes, anything in the past